### PR TITLE
chore(deps): upgrade to yarn v3.2.0

### PR DESCRIPTION
**Pull request by thisislawatts**
PR Checklist

- [X] - Commit [messages are descriptive](https://github.com/ONEARMY/community-platform/blob/master/CONTRIBUTING.md#--commit-style-guide), it will be used in our [Release Notes](https://github.com/ONEARMY/community-platform/releases/)
- [X] - Latest `master` branch merged

PR Type

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [X] Developer experience (improves developer workflows for contributing to the project)

## Description

Further information: https://dev.to/arcanis/yarn-32-libc-yarn-explain-next-major--o22

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -37,8 +37,8 @@ aliases:
         # https://circleci.com/docs/2.0/yarn/
         # when lock file changes, use increasingly general patterns to restore cache (also need to change `save_cache` path)
         # NOTE - if changing base image or received cache permission denied may need to bump version v3->v4 or similar (clears after 15d)
-        - yarn-packages-v12-{{ checksum "yarn.lock" }}
-        - yarn-packages-v12-
+        - yarn-packages-v12.3-{{ checksum "yarn.lock" }}
+        - yarn-packages-v12.3-
```
Comment (chrismclarke) [inline .circleci/config.yml:41]: Given that this is just an arbitrary string, just wondering why `12.3` as opposed to any other system? Does the `3` represent yarn 3?

**Review by chrismclarke — APPROVED** [review]
Looks good to me, added a quick question about cache naming convention (something which I'm sure was originally just copy-paste anyway so not hugely important/blocking), but happy to approve in any case

**Comment by thisislawatts** [comment]
Thanks for taking a look so quickly @chrismclarke. The cache naming was me testing out the build failures I've been seeing, pulling that out to a separate branch https://github.com/ONEARMY/community-platform/pull/1559

🤞🏻 This passes

**Pull request merged**